.goreleaser.yml

```
@@ -58,6 +60,41 @@ dockers:
       - ".gitignore"
       - "cmd"
       - "pkg"
+  - goos: linux
+    goarch: arm64
+    image_templates:
+      - "ghcr.io/philips-labs/{{ .ProjectName }}:v{{ .Version }}-arm64"
+      - "ghcr.io/philips-labs/{{ .ProjectName }}:{{ .FullCommit }}-arm64"
+    use: buildx
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/arm64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.description={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
```

We missed this one before:
```suggestion
      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
      - "--label=org.opencontainers.image.source={{.GitURL}}"
```